docs/DEPLOYING.md

```diff
@@ -48,6 +48,22 @@ human-readable or structured JSON log format will be chosen automatically. This
 can be overridden with the `force_json_output` and `stackdriver_json_output`
 configuration parameters under `logging_config`.
 
+The environment variable can be overridden at runtime using the `/traceconfigz`
```

Comment:
For clarity, since the last sentence changed topics
```suggestion
The `RUST_LOG` environment variable can be overridden at runtime using the `/traceconfigz`
```